src/App.test.js

```diff
@@ -1,9 +1,18 @@
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
-});
+// test('renders learn react link', () => {
+//   const { getByText } = render(<App />);
+//   const linkElement = getByText(/learn react/i);
+//   expect(linkElement).toBeInTheDocument();
+// });
+
+// describe('App', () => {
+//   it('should be able to render movies to the DOM', () => {
+//     const mockDisplayMovies = jest.fn();
```

Comment:
We'll talk about how to mock out isolated functions tomorrow in class. You need to make sure that the version of the function that the App component will import when running this test is going to be a mock.